packages/zowe-explorer/package.json

```diff
@@ -2092,36 +2094,54 @@
     "vscode": "^1.79.0"
   },
   "devDependencies": {
+    "@cucumber/cucumber": "9.5.1",
     "@napi-rs/cli": "^2.16.1",
     "@types/chai": "^4.2.6",
     "@types/chai-as-promised": "^7.1.0",
     "@types/expect": "^1.20.3",
     "@types/fs-extra": "^7.0.0",
     "@types/promise-queue": "^2.2.0",
+    "@types/sinon": "^17.0.3",
+    "@wdio/browser-runner": "^8.37.0",
+    "@wdio/cli": "^8.36.1",
+    "@wdio/cucumber-framework": "^8.36.1",
+    "@wdio/junit-reporter": "^8.36.1",
+    "@wdio/local-runner": "^8.36.1",
+    "@wdio/mocha-framework": "^8.37.0",
+    "@wdio/spec-reporter": "^8.36.1",
+    "@wdio/types": "^8.36.1",
     "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
     "chalk": "^2.4.1",
     "copy-webpack-plugin": "^12.0.2",
     "eslint-plugin-zowe-explorer": "3.0.0-next-SNAPSHOT",
     "expect": "^24.8.0",
+    "expect-webdriverio": "^4.13.0",
     "glob": "^7.1.6",
     "jest-silent-reporter": "^0.5.0",
     "log4js": "^6.4.6",
     "markdownlint-cli": "^0.33.0",
-    "sinon": "^6.1.0"
+    "sinon": "^16.1.1",
+    "ts-node": "^10.9.2",
+    "wdio-vscode-service": "^6.0.3",
+    "wdio-wait-for": "^3.0.11",
+    "webdriverio": "^8.36.1"
   },
   "dependencies": {
     "@zowe/core-for-zowe-sdk": "8.0.0-next.202404032038",
     "@zowe/secrets-for-zowe-sdk": "8.0.0-next.202404032038",
     "@zowe/zos-files-for-zowe-sdk": "8.0.0-next.202404032038",
     "@zowe/zos-jobs-for-zowe-sdk": "8.0.0-next.202404032038",
     "@zowe/zosmf-for-zowe-sdk": "8.0.0-next.202404032038",
-    "@zowe/zowe-explorer-api": "3.0.0-next-SNAPSHOT",
+    "@zowe/zowe-explorer-api": "workspace:3.0.0-next-SNAPSHOT",
     "dayjs": "^1.11.10",
     "fs-extra": "8.0.1",
     "isbinaryfile": "4.0.4",
     "promise-queue": "2.2.5"
   },
+  "peerDependencies": {
+    "@cucumber/cucumber": "9.5.1"
```

Comment:
Do we need Cucumber as both a dev dep and a peer dep?

Reply:
Nope, I think I tried this alongside with some other changes in an attempt to resolve an issue with Cucumber, but I forgot to remove this after discovering the solution. I've addressed this, in addition to your other suggestions, in commit d43970e.